docs/GetStarted/buildRPC.md

```diff
@@ -58,6 +70,9 @@ Developed in Go* programming language, the Remote Provisioning Client (RPC) appl
 2. Confirm the version numbers.
 
 ## Get the RPC
+If you have already cloned the toolkit repository as described in [Set Up](setup.md), change to the cloned `rpc-go` directory and see [Build the RPC](#buildtherpc). 
```

Comment:
You don't actually need to reference an anchor, the markdown headings have this built in, you can reference (#build-the-rpc)